analyses/fusion_filtering/run_fusion_filtering.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# K S Gaonkar 
+
+# Run fusion_filtering
+
+set -e
+set -o pipefail
+
```

Comment:
Yeah good idea to set up variables for files and and long format param values!! Definitely will help in giving the correct values. :/

Reply:
I've found it's often about making it harder to do something you did not intend to do or harder give the incorrect values because there are of course limits to human attention and memory 🙂